fitbit-react/src/Pages/Homepage/FeaturedCategories/FeaturedCategories.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { useFilter } from "../../../Context/filter-context";
+import NewArrivalCard from "../NewArrivals/NewArrivalCard";
+import "./featuredcategories.css";
+
+function FeaturedCategories() {
+  const { dispatch } = useFilter();
```

Comment:
I suggest it would be better if you name it more descriptive than dispatch, if you are using multiple Context and Reducers with all of them having "dispatch" as name of dispatch function, it might cause problems or confusion.